.github/workflows/install-sh.yml

```diff
@@ -5,57 +5,12 @@ on:
   workflow_dispatch:
 
 jobs:
-
-  sync-install:
-    runs-on: ubuntu-20.04
-    timeout-minutes: 10
-    steps:
-      - name: (PR comment) Get PR branch
-        if: ${{ github.event_name == 'issue_comment' }}
-        uses: xt0rted/pull-request-comment-branch@v2
-        id: comment-branch
-
-      - name: (PR comment) Set latest commit status as pending
-        if: ${{ github.event_name == 'issue_comment' }}
-        uses: myrotvorets/set-commit-status-action@v2.0.1
-        with:
-          sha: ${{ steps.comment-branch.outputs.head_sha }}
-          token: ${{ secrets.GITHUB_TOKEN }}
-          context: Update install.sh Suite
-          status: pending
-
-      - name: (PR comment) Checkout PR branch
-        if: ${{ github.event_name == 'issue_comment' }}
-        uses: actions/checkout@v3
-        with:
-          ref: ${{ steps.comment-branch.outputs.head_ref }}
-
-      - name: Checkout branch
-        if: ${{ github.event_name != 'issue_comment' }}
-        uses: actions/checkout@v3
-
-      - run: pip3 install awscli --upgrade --user
-      - env:
-          AWS_ACCESS_KEY_ID: "${{ secrets.CI_AWS_ACCESS_KEY_ID }}"
-          AWS_SECRET_ACCESS_KEY: "${{ secrets.CI_AWS_SECRET_ACCESS_KEY }}"
-        run: make sync-install
-
-      - name: (PR comment) Set latest commit status as failed
-        uses: myrotvorets/set-commit-status-action@v2.0.1
-        if: failure() && github.event_name == 'issue_comment'
-        with:
-          sha: ${{ steps.comment-branch.outputs.head_sha }}
-          token: ${{ secrets.GITHUB_TOKEN }}
-          context: Update install.sh Suite
-          status: 'failure'
-
   test-install:
-    needs: sync-install
     runs-on: ubuntu-20.04
     timeout-minutes: 5
     steps:
       - run: sudo apt-get install --yes curl bc
-      - run:  curl --proto '=https' --tlsv1.2 -sSfL https://sh.vector.dev | bash -s -- -y
+      - run: bash distribution/install.sh -- -y
```

Comment:
Purely curiosity- why is the explicit `bash` call needed?

Reply:
Ah, given the file does have the executable bit (`x`) it technically isn't needed. I just have a habit of using `bash` with anything ending in `.sh` since those frequently aren't executable.